test/realtime/block_waiver_store_test.exs

```diff
@@ -0,0 +1,62 @@
+defmodule Realtime.BlockWaiverStoreTest do
+  use ExUnit.Case, async: true
+
+  alias Realtime.{BlockWaiver, BlockWaiverStore}
+
+  @block_waivers [
+    %BlockWaiver{
+      start_time: 10,
+      end_time: 20,
+      remark: "E:1106"
+    }
+  ]
+  @block_waivers_by_block_and_service_ids %{
+    {"block1", "service1"} => @block_waivers
+  }
+
+  describe "start_link/1" do
+    test "starts up and lives" do
+      {:ok, server} = BlockWaiverStore.start_link(name: :start_link)
+
+      Process.sleep(10)
+
+      assert Process.alive?(server)
+    end
+  end
+
+  describe "block_waivers_for_block_and_service/2" do
```

Comment:
I think this could also use a test for the case where we don't have a block waiver for the block.